modules/tensor_mechanics/tests/generalized_plane_strain/tests

```diff
@@ -1,10 +1,26 @@
 [Tests]
```

Comment:
@bwspenc should we add those tests? If so, this can be addressed in another PR. @jiangwen84